asv_bench/benchmarks/algorithms.py

```diff
@@ -174,4 +174,14 @@ def time_argsort(self, N):
         self.array.argsort()
 
 
+class SortIndexSeries:
+    def setup(self):
+        N = 10 ** 5
+        idx = pd.date_range(start="1/1/2000", periods=N, freq="s")
+        self.s = pd.Series(np.random.randn(N), index=idx)
+
+    def time_sort_index(self):
+        self.s.sort_index()
+
+
```

Comment:
I don't think we need this. SortIndex.time_sort_index already covers this code path.

Reply:
Right, I haven't checked `timeseries` before adding asv